ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantTestContext.scala

```diff
@@ -150,6 +155,10 @@ private[testtool] final class ParticipantTestContext private[participant] (
   val nextKeyId: () => String = nextIdGenerator("key")
   val nextUserId: () => String = nextIdGenerator("user", lowerCase = true)
 
+  val delayMechanism: DelayMechanism = if (features.staticTime) {
+    new StaticTimeDelayMechanism(this)
+  } else new TimeDelayMechanism()
```

Comment:
I am a big fan of keeping branches on the same indentation level.

```suggestion
  val delayMechanism: DelayMechanism =
    if (features.staticTime)
    new StaticTimeDelayMechanism(this)
  else
    new TimeDelayMechanism()
```

Reply:
Also maybe a `lazy val`, just because we usually won't use it?

Reply:
Changed both. 
Weird about the indentation, I much prefer it on the same level as well, but somehow was expecting the auto formatter to set it properly and I didn't check it.